### PR TITLE
edencommon fbthrift fizz folly wangle watchman 2023.03.06.00

### PR DESCRIPTION
**Pull request by ZhongRuoyu**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Still no `fb303` release: https://github.com/facebook/fb303/tags

### CODE REVIEW

**Review by SMillerDev — APPROVED** [review]

File: Formula/fizz.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Fizz < Formula
   desc "C++14 implementation of the TLS-1.3 standard"
   homepage "https://github.com/facebookincubator/fizz"
-  url "https://github.com/facebookincubator/fizz/releases/download/v2023.02.27.00/fizz-v2023.02.27.00.tar.gz"
+  url "https://github.com/facebookincubator/fizz/releases/download/v2023.03.06.00/fizz-v2023.03.06.00.tar.gz"
   sha256 "54d91581da093d70cc0baf2a391c0bd64b6015dc70c90c1111c2a4d40240f5f7"
```
Comment (carlocab) [inline Formula/fizz.rb:5]: Same checksum? Huh.

**Comment by ZhongRuoyu** [comment]
As usual, CI on Intel Big Sur was [canceled](https://github.com/Homebrew/homebrew-core/actions/runs/4342374979/jobs/7583233268). But it seems that the job actually finished; the bottles are all there.

**Review by carlocab — APPROVED** [review]

**Comment by ZhongRuoyu** [comment]
Huh, `fb303` now has a `v2023.03.06.00`: https://github.com/facebook/fb303/tags

But this week there's no release from any of the Facebook formulae (yet). If that remains the case, I guess we can bump `fb303`, and consider add it back to `synced_versions_formulae.json`.

**Comment by carlocab** [comment]
Let's bump it and add it back. We can just remove it if the tag is delayed again.

**Pull request closed without merging**